### DISCUSSION TITLE
How to eliminate the border gap with border none

### DISCUSSION THREAD

**Discussion by kaisiant** [diegomura/react-pdf#2043] [Q&A]
![image](https://user-images.githubusercontent.com/26655111/194462029-5487c393-bfae-4209-a751-01c5ca91fe6e.png)

As you can see, there are some gaps between the rows. Does anyone have any idea about this?

**Comment by krishna-404** [comment]
Recently ran into this. Couldnt find a work around. 

PS: For now, I did put everything into a parent component & gave it a border, where ever necessary & possible.
